src/client/utils/constants.ts

```diff
@@ -27,3 +27,8 @@ export const notesSortOptions = [
   { value: NotesSortKey.LAST_UPDATED, label: 'Last Updated' },
   { value: NotesSortKey.FAVORITES, label: 'Favorites' },
 ]
+
+export const directionTextOptions = [
```

Comment:
Typo `rifht`. The labels should be "Left to right" and "Right to left", not `rifht-to-left` with hyphens.

Reply:
done